src/Voron/Impl/Journal/WriteAheadJournal.cs

```diff
@@ -123,6 +123,9 @@ private JournalFile NextFile(int numberOf4Kbs = 1)
             if (_currentJournalFileSize < minRequiredSize)
             {
                 _currentJournalFileSize = Bits.NextPowerOf2(minRequiredSize);
+                if (_currentJournalFileSize > _env.Options.MaxLogFileSize)
+                    _currentJournalFileSize = Math.Max(_env.Options.MaxLogFileSize, minRequiredSize);
```

Comment:
I think this can be changed to:
_currentJournalFileSize = minRequiredSize;
since in this block:
1. _currentJournalFileSize  > _env.Options.MaxLogFileSize
1. _currentJournalFileSize  < minRequiredSize
thus
minRequiredSize >  _currentJournalFileSize   > _env.Options.MaxLogFileSize => 
minRequiredSize >_env.Options.MaxLogFileSize

Reply:
Assume `minRequiredSize` is 33MB in size and `MaxLogFileSize` is 48MB
So `_currentJournalFileSize` is 64MB initially and is larger than `MaxLogFileSize`

We want to set it to the 48MB rather than 33MB in this case, because we want to reduce the number of allocation on the FS.